src/_layouts/default.html

```diff
@@ -3,13 +3,20 @@
 <!doctype html>
 <html lang="en">
 <head>
+	<title>Tabler Icons</title>
+    <meta content="Tabler Icons" property="og:title" />
+    <meta content="Over 2400 pixel-perfect icons for web design" property="og:description" />
+    <meta content="https://tabler-icons.io/" property="og:url" />
+    <meta content="https://i.imgur.com/3dzpXuc.png" property="og:image" />
+    <meta content="#1f6bc4" data-react-helmet="true" name="theme-color" />
```

Comment:
Where is this color coming from, because I cannot find [on the color palette](https://preview.tabler.io/docs/colors.html)?

Snippet:
```
--tblr-blue: #206bc4;
--tblr-blue-fg: var(--tblr-light);
--tblr-blue-darken: #1d60b0;
...
--tblr-tabler: #206bc4;
--tblr-tabler-fg: var(--tblr-light);
--tblr-tabler-darken: #1d60b0;
```